### ISSUE TITLE
 Support for Content-Security Policy

### ISSUE THREAD

**Issue by pfcloutier-druide** [karser/KarserRecaptcha3Bundle#20]
Hello,

 For supporting website with Content-Security Policy, we must include a nonce for inline JavaScript. I would like to suggest a new parameter : 

```
$builder->add('captcha', Recaptcha3Type::class, [
    'constraints' => new Recaptcha3(),
    'action_name' => 'homepage',
    'nonceCSP' => $nonceCSP
]);
```

so it can be used in : 

```

<script nonce="{{ nonceCSP }}">
    var recaptchaCallback_form_captcha = function() {
    grecaptcha.execute('<YOUR-RECAPTCHA-KEY>', {action: 'landing'}).then(function(token) {
    document.getElementById('form_captcha').value = token;
    });
    };
    </script>
```

Thank you!

**Comment by karser** [comment]
Hello @pfcloutier-druide Looks like this feature is useful. Do you want to send a PR?

**Comment by pfcloutier-druide** [comment]
I made a PR : https://github.com/karser/KarserRecaptcha3Bundle/pull/21

**Comment by karser** [comment]
Merged and released in v0.1.8
